daml-lf/archive/BUILD.bazel

```diff
@@ -139,7 +139,7 @@ daml_compile(
 daml_compile(
     name = "DarReaderTest-dev",
     main_src = "src/test/daml/DarReaderTest.daml",
-    target = "1.dev",
+    target = "1.6",
```

Comment:
Please do this instead:

https://github.com/digital-asset/daml/blob/5daf3f55f31ab327be04031c2d39c48b80cc7c0f/daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala#L62-L65

Reply:
If you don't specify a `target` at all, you get the newest stable version of DAML-LF. I think that is the right to do in almost all cases, except for when you want to test something specific to an older version.

Reply:
Here we want to test something in a latest version (`1.6`) which is still not stable.

Reply:
@remyhaemmerle-da I'm not sure I can follow you here. Once 1.6 is frozen, it is the latest stable version. If you don't specify a target, the compiler will produce 1.6 once this PR is merge and until 1.7 is frozen.

Reply:
Is it not the  `versionDefault` that defines the default output of the compiler ?
https://github.com/digital-asset/daml/blob/7bad280b280bb4240dca7b6b1d670076d13dc07f/compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs#L32

I did not change it, so it is sill `1.5`. Should I ?

Reply:
Yes, please change it. I completely missed that. Apologies for the confusion.

Reply:
fixed

Reply:
Thanks a lot.